include/drivers/misc/ft8xx/ft8xx.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Hubert Miś
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief FT8XX public API
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISPLAY_FT8XX_FT8XX_H_
+#define ZEPHYR_DRIVERS_DISPLAY_FT8XX_FT8XX_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct ft8xx_touch_transform {
```

Comment:
public API should be documented (data structures, functions, etc.). there should be a new doxygen group for misc drivers and place each driver into its own group.